data/tools/extract-wikidata.py

```diff
@@ -0,0 +1,379 @@
+"""
+Extracts lexicographic data from Wikidata for Unicode Inflection usage
+
+First you need to download the lexicographic data wholesale from Wikidata.
+You can do this by downloading the data from the Web, with either of the
+following two commands.
+
+The first one downloads a specifc version of the data. The following
+command downloads the first version of 2025:
+
+wget -O lexemes.json.bz2 https://dumps.wikimedia.org/wikidatawiki/entities/20250101/wikidata-20250101-lexemes.json.bz2
+
+A list of all available versions can be found on the Web here:
+https://dumps.wikimedia.org/wikidatawiki/entities/
+
+Or you can download the most recent version using the following command:
+
+wget -O lexemes.json.bz2 https://dumps.wikimedia.org/wikidatawiki/entities/latest-lexemes.json.bz2
+
+After that run this script in the dictionary in which the lexemes.json.bz2 file is located.
+The result will be a dictionary_xxx.lst file for the given language.
+Per default the language is 'en', but you can use one of the permitted language code insteads.
```

Comment:
Maybe:
Language can be specified in command line, default is 'en'. Allowed languages are listed below in code2qid object.